### DISCUSSION TITLE
How to recover my blocked admin account?

### DISCUSSION THREAD

**Discussion by chikhladzee** [appwrite/appwrite#5260] [Q&A] [answered]
**_The current User has been blocked.You can unblock the user from the Appwrite console._**

i deleted my account,i thought i would make a new one for the nex new project with the same email..but i have this error..how can i reset or delete and start over? i read similar question here but i couldn;t undestand it, there was nothing mentioned about how to do it,it was just explained why it is happening.that it for protection... i'm beginner and it's a bit difficult for me to understand, can you help me,where can i do this ? thanks

**Comment by stnguyen90** [comment, marked as answer]
The only way to recover the account is to go directly into the database and update the user status. You can try running the following after [backing up your data](https://youtube.com/watch?v=lM5yZEPtlvg):

```shell
docker exec -it appwrite-mariadb sh -c 'mysql -uroot -p$MYSQL_ROOT_PASSWORD $MYSQL_DATABASE -e "UPDATE _console_users SET status = 1 WHERE email = \"YOUR@EMAIL.COM\""'
```

Make sure to replace `YOUR@EMAIL.COM` with your email.

**Reply by chikhladzee** [reply]
thanks

**Comment by Soham-Soham** [comment]
I also tried to delete my account and then again try to create/login account but it showing ( The current user has been blocked. You can unblock the user by making a request to the User API's "Update User Status" endpoint or in the Appwrite Console's Auth section .)
As I'm a beginner so don't understand what to do
